Src/BootCamp.Chapter/Credentials.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    // TODO: make a struct and add validation and other needed methods (if needed)
```

Comment:
Codacy found an issue: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5768165)